extensions/resource-deployment/src/ui/deploymentInputDialog.ts

```diff
@@ -91,6 +115,6 @@ export class DeploymentInputDialog extends DialogBase {
 	}
 
 	private executeNotebook(notebookDialogInfo: NotebookBasedDialogInfo): void {
-		this.notebookService.backgroundExecuteNotebook(notebookDialogInfo.taskName, notebookDialogInfo.notebook, 'deploy', this.platformService);
+		this.notebookService.backgroundExecuteNotebook(notebookDialogInfo.taskName, <string | NotebookPathInfo>notebookDialogInfo.notebook, 'deploy', this.platformService);
```

Comment:
<> is deprecated, use as

```
(notebookDialogInfo.notebook as string | NotebookPathInfo)
```

Reply:
Amir can you share a pointer to where it says it is deprecated? From the official typescript doc, I see that both are acceptable and equivalent.
https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions

The newer 'as' was created to be friendly with jsx so it works in all environments.